plugins/modules/aci_aaa_user_certificate.py

```diff
@@ -37,7 +37,6 @@
     description:
     - The name of the user certificate entry in ACI.
     type: str
-    aliases: [ cert_name ]
```

Comment:
Add to the spec instead?

Reply:
It conflicts with global certificate_name option that define cert_name as an alias